learning_assistant/views.py

```diff
@@ -35,6 +38,27 @@ class CourseChatView(APIView):
     authentication_classes = (SessionAuthentication, JwtAuthentication,)
     permission_classes = (IsAuthenticated,)
 
+    def __save_user_interaction(self, user_id, user_message, assistant_message):
```

Comment:
can this be moved to the `api.py` file? I also think it might make more sense for it to look like `save_user_message(user_id, role, message)`, and call it twice from within the view.

Reply:
Moved to the API module and added two calls in the view.